src/arch/src/aarch64/fdt.rs

```diff
@@ -660,7 +660,14 @@ mod tests {
     #[test]
     fn test_create_fdt_with_devices() {
         let regions = arch_memory_regions(layout::FDT_MAX_SIZE + 0x1000);
-        let mem = GuestMemoryMmap::from_ranges(&regions).expect("Cannot initialize memory");
+        let mem = vm_memory::create_guest_memory(
+            &regions
+                .into_iter()
+                .map(|r| (None, r.0, r.1))
+                .collect::<Vec<_>>(),
+            false,
+        )
+        .expect("Cannot initialize memory");
```

Comment:
The only difference for this test helper would be that it had a parameter `regions: &[(GuestAddress, usize)]` instead of `regions: &[(Option<FileOffset>, GuestAddress, usize)]`.
I cannot think of a good name for such a function but if you have a good suggestion, please do tell.
I also think that for such a small argument difference, a separate function may be overkill, WDYT?

Reply:
What about `create_anonymous_guest_memory()` ? Since it's not file-backed. I think it would make sense having this separate method since now we duplicate the following code segment quite a lot.

```
            &regions
                .into_iter()
                .map(|r| (None, r.0, r.1))
                .collect::<Vec<_>>(),
```

Reply:
sounds fair, will do

Reply:
done